object_storage_api/.env.example

```
@@ -20,4 +20,5 @@ OBJECT_STORAGE__SECRET_ACCESS_KEY=example_password
 OBJECT_STORAGE__BUCKET_NAME=object-storage
 OBJECT_STORAGE__PRESIGNED_URL_EXPIRY_SECONDS=1800
 ATTACHMENT__MAX_SIZE_BYTES=104857600
+IMAGE__ALLOWED_MIME_TYPES=["image/jpeg", "image/png", "image/bmp", "image/tiff", "image/webp"]
```

```suggestion
IMAGE__ALLOWED_MIME_TYPES=["image/jpeg", "image/png", "image/bmp", "image/tiff", "image/webp", "image/heic", "image/heif"]
```
IPhones released after 2017 store images as a HEIC/HEIF file by default instead of JPEG. They do automatically convert to JPEG if you share them using some methods (like email), but stay as a HEIC/F if you upload them (like using Google Drive). 

If they're using their phones to take photos and upload them this could pose a problem. I think most android phones use JPEG as their default; I think some support HEIC, but I'm not sure if it's set as default or not.

---

I can't see this on the [page](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html) about supported file formats by Pillow so I am assuming it will just break when it tries to open an image of this type?

---

Good spot, I think I have an image somewhere I can try.